UPGRADE.md

```diff
@@ -1,3 +1,7 @@
+# Upgrading from v2.0.1 to v2.1.0
+
+* the internal class src/Util/FilenameUtils.php has been removed.
```

Comment:
the name of the class should be like `this`

Reply:
Adapted documentation accordingly.